test/e2e/serial/tests/workload_placement.go

```diff
@@ -1017,18 +1016,8 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 
 				rl := e2ereslist.FromReplicaSet(*rs)
 
-				dataBefore, err := yaml.Marshal(nrtInitialTarget)
-				Expect(err).ToNot(HaveOccurred())
-				dataAfter, err := yaml.Marshal(updatedTargetNrt)
-				Expect(err).ToNot(HaveOccurred())
-				match, err := e2enrt.CheckNodeConsumedResourcesAtLeast(*nrtInitialTarget, *updatedTargetNrt, rl, podQoS)
-				Expect(err).ToNot(HaveOccurred())
-
-				if match == "" {
-					klog.Warningf("inconsistent accounting: no resources consumed by the running pod,\nNRT before test replicaset: %s \nNRT after: %s \npod resources: %v", dataBefore, dataAfter, e2ereslist.ToString(rl))
-					return false
-				}
-				return true
+				_, match := checkNRTConsumedResources(fxt, *nrtInitialTarget, rl, &pods[0])
```

Comment:
checkNRTConsumedResources (now) checks resources consumption on numa zones. need to be cautious when using this function to check the NRT getting updated especially if we are checking pod scope. This test for example is anticipated to run on both scopes. Assuming it run on pod scope, it runs 10 replicas (each with 2 containers), using checkNRTConsumedResources to check the NRT updates will check if the total requested resources are consumed from one numa, which is not accrate because we are testing pod scope, and the scheduler may place different containers on different numas.

Reply:
I'll anyway run this test from on my env to better understand the behavior.

Reply:
This is a good point. To save time I just run container scope, so this is likely to be still broken (or more broken!) on pod scope.

Reply:
I'll verify on pod scope then